widget/hyperlink_test.go

```diff
@@ -0,0 +1,53 @@
+package widget
+
+import (
+	"net/url"
+	"testing"
+
+	"fyne.io/fyne"
+	_ "fyne.io/fyne/test"
+	"fyne.io/fyne/theme"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHyperlink_MinSize(t *testing.T) {
+	hyperlink := NewHyperlink("Test", "url1")
+	min := hyperlink.MinSize()
+
+	assert.True(t, min.Width > theme.Padding()*2)
+
+	hyperlink.SetText("Longer")
+	assert.True(t, hyperlink.MinSize().Width > min.Width)
+}
+
+func TestHyperlink_Alignment(t *testing.T) {
+	hyperlink := &Hyperlink{Text: "Test", Alignment: fyne.TextAlignTrailing}
+	assert.Equal(t, fyne.TextAlignTrailing, Renderer(hyperlink).(*textRenderer).texts[0].Alignment)
```

Comment:
There is a helpful shortcut for testing "textRenderTests" that you can pass "hyperlink" and index with [0]:

textRenderTexts(hyperlink)[0].Alignment

Reply:
Changed in 3fdc033528934b7e8ed483474c570a7f82cbcc9a